app/src/main/res/values-ja/localized.xml

```diff
@@ -1,2 +1,424 @@
+    <string name="p_skip_servant_face_checks">サーヴァント画像チェックをスキップ</string>
+    <string name="p_skip_servant_face_checks_summary">Brave chains、サーヴァント優先度、NP一致 カードシャッフル、などが機能しません</string>
+
+    <string name="p_wait_ap_regen_text">AP回復を待つ</string>
+    <string name="wait_ap_regen_toast_message">AP回復を待っています。 %d 分後に再確認します。</string>
+
+    <string name="p_storage">ストレージ</string>
+    <string name="p_folder">サポート画像、ドロップ画像などに使用するフォルダを変更する。</string>
+
+    <string name="p_choose_folder_title">フォルダ選択</string>
+    <string name="p_choose_folder_message">サポート画像やドロップ画像などを保存するフォルダを選択します。\n\n以前のインストールから既にファイルがある場合は、「support」フォルダではなく、その親フォルダを選んでください。\n\n注意：デバイスのファイルピッカーで「内部ストレージを表示」を使用する必要があるかもしれません。</string>
+    <string name="p_choose_folder_not_exist_title">他のフォルダーを選択する</string>
+    <string name="p_choose_folder_not_exist_message">サポート画像やドロップ画像などを保存するフォルダにアクセスできません。</string>
+    <string name="p_choose_folder_action">選択</string>
+
+    <string name="p_advanced">その他</string>
+    <string name="p_debug_mode">デバッグモード</string>
+    <string name="p_debug_mode_summary">一致した箇所をハイライトします</string>
+    <string name="p_record_screen">画面録画</string>
+    <string name="p_record_screen_summary">画面録画し \'record.mp4\' を出力フォルダに保存します。\n複製か名前を変更して保存してください。</string>
+    <string name="p_root_screenshot">Root権限でスクリーンショットを撮影する</string>
+    <string name="p_root_screenshot_summary">Nox (Android 7)などのエミュレータで使用します。有効にした後、一度停止させ、再度実行してください。 </string>
+    <string name="p_auto_start_service">自動でサービス開始</string>
+    <string name="p_thresholded_stage_counter">ウェーブカウンターのしきい値を検出</string>
+
+    <string name="p_fine_tune">微調整</string>
+    <string name="p_fine_tune_support">サポート</string>
+    <string name="p_fine_tune_support_swipes_per_update">更新までのスワイプ回数</string>
+    <string name="p_fine_tune_support_max_updates">更新回数上限</string>
+    <string name="p_fine_tune_similarity">類似度</string>
+    <string name="p_fine_tune_min_similarity">最小の類似度</string>
+    <string name="p_fine_tune_mlb_similarity">MLBの類似度</string>
+    <string name="p_fine_tune_stage_counter_similarity">ウェーブカウンターの類似度</string>
+    <string name="p_fine_tune_clicks">クリック</string>
+    <string name="p_fine_tune_wait_after_clicking">クリック後の待機秒数</string>
+    <string name="p_fine_tune_click_duration">クリックの継続時間</string>
+    <string name="p_fine_tune_click_delay">マルチクリック間隔</string>
+    <string name="p_fine_tune_swipes">スワイプ</string>
+    <string name="p_fine_tune_wait_after_swiping">スワイプ後の待機秒数</string>
+    <string name="p_fine_tune_swipe_duration">スワイプの継続時間</string>
+    <string name="p_fine_tune_swipe_multiplier">スワイプ時間倍率</string>
+    <string name="p_fine_tune_wait">待機</string>
+    <string name="p_fine_tune_skill_delay">スキル使用後の待機時間</string>
+    <string name="p_fine_tune_wait_before_turn">ターン経過後の待機時間</string>
+    <string name="p_fine_tune_wait_before_cards">カード選択後の待機時間</string>
+    <string name="p_fine_tune_wait_multiplier">待機時間倍率</string>
+
+    <string name="troubleshoot">Troubleshooting</string>
```

Comment:
Although this can also be translated directly, it is a notation that frequently appears in foreign-made applications, and it would be more helpful to leave it as it is.